internal/archive/module_location.go

```diff
@@ -22,6 +21,15 @@ func (ep *ModuleLoc) GetTargetTmpDir() string {
 	return ep.loc.GetTarget()
 }
 
+// GetTargetTmpDir - gets the target root path;
+func (ep *ModuleLoc) GetTargetTmpRoot() string {
+	if ep.targetPathDefined {
+		return ep.loc.GetTarget()
+	}
+	// default target folder for module build results defined in the temp folder
```

Comment:
Maybe this will make it easier to understand the use case:
```suggestion
	// default target folder for module build results is defined under the temp folder
```

Reply:
changed